### PR TITLE
Refactor CSS (some pages and components)

### PR DESCRIPTION
**Pull request by Publicker**
This is a start of #3504 

I need to know if I am doing it right and know if is another better way to do it 🐱‍🏍. 
I will be waiting for some feedback!

### CODE REVIEW

**Comment by mittalyashu** [comment]
Your changes also include formating and indentation. Can you also run `yarn run lint:fix` to keep it consistent?

**Review by mittalyashu — CHANGES_REQUESTED** [review]

File: src/renderer/components/input/inputText.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
 			...borderProps,
 			...fontProps
 		}"
-		@input="$emit('input', $event.target.value)"
+		@input="mit('input', $event.target.value)"
```
Comment (mittalyashu) [inline src/renderer/components/input/inputText.vue]:
```suggestion
		@input="$emit('input', $event.target.value)"
```

File: src/renderer/pages/repository/workspace.vue [hunk 2]
```diff
@@ -223,11 +215,12 @@ export default {
 
 <style lang="sass">
 .workspace
-	&__files
-		border-right: 1px solid #DEE0E3
-		width: 300px
+	border-right: 1px solid #DEE0E3
+	width: 300px
+	max-width: 300px;
+	min-width: 300px;
```
Comment (mittalyashu) [inline src/renderer/pages/repository/workspace.vue]:
```suggestion
  max-width: 300px
	min-width: 300px
```

**Review by mittalyashu — CHANGES_REQUESTED** [review]

**Comment by mittalyashu** [comment]
Can you also remove all the transition related styles? We do have plans to animations and transition but not right now.

File: src/renderer/components/TLayouts/TScrollbar.vue [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <template>
-	<vue-scrollbar :style="{ height: height, width: width }">
+	<vue-scrollbar
+		:style="{ height: height, width: width, margin: 0, padding: padding }"
```
Comment (mittalyashu) [inline src/renderer/components/TLayouts/TScrollbar.vue]: What is the idea behind using padding as props?

File: src/renderer/pages/repository/workspace.vue [hunk 4]
```diff
@@ -1,12 +1,8 @@
...
-		<div ref="workspaceFiles" class="workspace__files">
-			<t-flexbox flex-direction="column" style="overflow: hidden;">
-				<t-flexbox
-					ref="branchName"
-					align-items="center"
-					class="workspace__branch"
-				>
+		<div ref="workspaceFiles" class="workspace">
+			<t-flexbox flex-direction="column" class="overflow-hidden">
+				<t-flexbox ref="branchName" align-items="center" class="branch">
```
Comment (mittalyashu) [inline src/renderer/pages/repository/workspace.vue]: Isn't `branch` class name too common to use?

File: src/renderer/components/commit/commitHistoryItem.vue [hunk 5]
```diff
@@ -94,13 +94,14 @@ export default {
 <style lang="sass">
 .history
 	&__item
-		padding: 10px
+		padding: 14px
 		font-size: 12px
-		border-bottom: 1px solid #DEE0E3
+		margin-bottom: 0.8rem
```
Comment (mittalyashu) [inline src/renderer/components/commit/commitHistoryItem.vue]: Can you revert back these changes?

**Review by mittalyashu — CHANGES_REQUESTED** [review]

File: src/renderer/components/TLayouts/TScrollbar.vue (hunk 3 above)
Comment (Publicker) [reply src/renderer/components/TLayouts/TScrollbar.vue]: I was doing a test and I forgot it. I just removed the prop now!

**Review by mittalyashu — CHANGES_REQUESTED** [review]

File: src/renderer/components/navbar.vue [hunk 6]
```diff
@@ -246,5 +246,5 @@ export default {
 
 		.navbar__item
 			&:not(:last-child)
-				margin-right: 15px
+				margin-right: 1.5rem
```
[the remark this answers is not in the document]
Comment (Publicker) [reply src/renderer/components/navbar.vue]: I see! 
I will reduce the margin!

File: src/renderer/components/sidebar.vue [hunk 7]
```diff
@@ -59,7 +59,7 @@ export default {
 
 		&__item
 			padding: 3px 20px
-			margin-bottom: 5px
+			margin-bottom: 1.5rem
```
[the remark this answers is not in the document]
Comment (Publicker) [reply src/renderer/components/sidebar.vue]: I will reduce this margin as well!

File: src/renderer/assets/css/_spacing.sass [hunk 8]
```diff
@@ -17,3 +17,19 @@
...
+	.ml-#{$i}
+		margin-left: $i + rem
+
+@for $i from 1 through 10
+	.mb-#{$i}
+		margin-bottom: $i + rem
+
+@for $i from 1 through 10
+	.mr-#{$i}
+		margin-right: $i + rem
```
Comment (mittalyashu) [inline src/renderer/assets/css/_spacing.sass]: Creating for loop for margins might take a lot of space in terms of file size and we can easily add those to class based css inside components files.

**Pull request closed without merging**